content/en/docs/contribute/style/write-new-topic.md

```diff
@@ -70,43 +71,47 @@ triple-dashed lines at the top of the page. Here's an example:
 
 Depending on your page type, put your new file in a subdirectory of one of these:
 
-* /docs/tasks/
-* /docs/tutorials/
-* /docs/concepts/
+* /content/en/docs/tasks/
+* /content/en/docs/tutorials/
+* /content/en/docs/concepts/
 
 You can put your file in an existing subdirectory, or you can create a new
 subdirectory.
 
-## Creating an entry in the table of contents
+## Placing your topic in the table of contents
 
-Depending page type, create an entry in one of these files:
+The table of contents is built dynamicaly using the directory structure of the
+documentation source. The top-level directories under `/content/en/docs/` create
+top-level navigation, and subdirectories each have entries in the table of
+contents.
 
-* /_data/tasks.yaml
-* /_data/tutorials.yaml
-* /_data/concepts.yaml
+Each subdirectory has a file `_index.md`, which represents the "home" page for
+a given subdirectory's content. The `_index.md` does not need a template. It
+can contain overview content about the topics in the subdirectory.
 
-Here's an example of an entry in /_data/tasks.yaml:
-
-    - docs/tasks/configure-pod-container/configure-volume-storage.md
+Other files in a directory are sorted alphabetically by default. This is almost
+never the best order. To control the relative sorting of topics in a
+subdirectory, set the `weight:` front-matter key to an integer. Typically, we
+use multiples of 10, to account for adding topics later. For instance, a topic
+with weight `10` will come before one with weight `20`.
 
 ## Including code from another file
```

Comment:
I have a PR under review for this sub-topic. #9574

Reply:
@tengqm Can you let me know when your PR is merged so I can make sure those changes get into this branch? I'm sure your PR will be ready before mine has been completely reviewed.

Reply:
I just merged it. Let me now rebase this PR and integrate that content into the new location.